packages/web3torrent/src/constants.ts

```diff
@@ -84,6 +100,32 @@ export const preSeededTorrents: Array<Partial<Torrent>> = [
   }
 ];
 
+export const preseededTorrentsUI: TorrentUI[] = preSeededTorrents.map(partialTorrent => ({
+  ...partialTorrent,
+  ...getStaticTorrentUI(partialTorrent.infoHash, partialTorrent.name, partialTorrent.length)
+}));
+
+export function getStaticTorrentUI(
```

Comment:
I don't know if I would put this function here..... it's not exactly a constant, isn't it?

Reply:
When I move `getStaticTorrentUI` into another module, I am running into into a circular dependency issue as `constant.ts` uses `getStaticTorrentUI` (and many other modules use the `constants` module). So I decided to move the function into constants. If you have other suggestions for getting around a circular dependency, I am all ears.